.travis.yml

```diff
@@ -0,0 +1,47 @@
+sudo: required
+dist: trusty
+matrix:
```

Comment:
>matrix [](start = 0, length = 6)

@ydy, can you include python in?

Reply:
https://github.com/codecov/example-python/blob/master/.travis.yml
https://docs.travis-ci.com/user/languages/python/#Default-Build-Script